sklearn/linear_model/tests/test_base.py

```diff
@@ -215,9 +218,11 @@ def test_linear_regression_sparse(random_state=0):
 @pytest.mark.filterwarnings("ignore:'normalize' was deprecated")
 @pytest.mark.parametrize("normalize", [True, False])
 @pytest.mark.parametrize("fit_intercept", [True, False])
-def test_linear_regression_sparse_equal_dense(normalize, fit_intercept):
+def test_linear_regression_sparse_equal_dense(
+    normalize, fit_intercept, global_random_seed
+):
     # Test that linear regression agrees between sparse and dense
-    rng = check_random_state(0)
+    rng = check_random_state(global_random_seed)
     n_samples = 200
```

Comment:
Test passes with all seeds. I experimented with different `n_samples`, which did not improve execution time. `n_samples` can be as small as `3` and test still passes with all seeds